third_party/sl4a/src/main/java/com/google/android/mobly/snippet/rpc/MethodDescriptor.java

```diff
@@ -270,327 +200,36 @@ public static Object buildIntent(JSONObject jsonObject) throws JSONException {
   }
 
-  public Method getMethod() {
-    return mMethod;
-  }
-
-  public Class<? extends Snippet> getDeclaringClass() {
-    return mClass;
-  }
-
   public String getName() {
-    if (mMethod.isAnnotationPresent(RpcName.class)) {
-      return mMethod.getAnnotation(RpcName.class).name();
-    }
     return mMethod.getName();
   }
 
   public Type[] getGenericParameterTypes() {
     return mMethod.getGenericParameterTypes();
   }
 
-  public Annotation[][] getParameterAnnotations() {
-    return mMethod.getParameterAnnotations();
-  }
-
   /**
    * Returns a human-readable help text for this RPC, based on annotations in the source code.
    *
    * @return derived help string
    */
   public String getHelp() {
-    StringBuilder helpBuilder = new StringBuilder();
-    Rpc rpcAnnotation = mMethod.getAnnotation(Rpc.class);
-
-    helpBuilder.append(mMethod.getName());
-    helpBuilder.append("(");
-    final Class<?>[] parameterTypes = mMethod.getParameterTypes();
-    final Type[] genericParameterTypes = mMethod.getGenericParameterTypes();
-    final Annotation[][] annotations = mMethod.getParameterAnnotations();
+    StringBuilder paramBuilder = new StringBuilder();
+    Class<?>[] parameterTypes = mMethod.getParameterTypes();
     for (int i = 0; i < parameterTypes.length; i++) {
-      if (i == 0) {
-        helpBuilder.append("\n  ");
-      } else {
-        helpBuilder.append(",\n  ");
+      if (i != 0) {
+        paramBuilder.append(", ");
       }
-
-      helpBuilder.append(getHelpForParameter(genericParameterTypes[i], annotations[i]));
-    }
-    helpBuilder.append(")\n\n");
-    helpBuilder.append(rpcAnnotation.description());
-    if (!rpcAnnotation.returns().equals("")) {
-      helpBuilder.append("\n");
-      helpBuilder.append("\nReturns:\n  ");
-      helpBuilder.append(rpcAnnotation.returns());
-    }
-
-    if (mMethod.isAnnotationPresent(RpcStartEvent.class)) {
-      String eventName = mMethod.getAnnotation(RpcStartEvent.class).value();
-      helpBuilder.append(String.format("\n\nGenerates \"%s\" events.", eventName));
-    }
-
-    if (mMethod.isAnnotationPresent(RpcDeprecated.class)) {
-      String replacedBy = mMethod.getAnnotation(RpcDeprecated.class).value();
-      String release = mMethod.getAnnotation(RpcDeprecated.class).release();
-      helpBuilder.append(String.format("\n\nDeprecated in %s! Please use %s instead.", release,
-          replacedBy));
+      paramBuilder.append(parameterTypes[i].getSimpleName());
     }
-
-    return helpBuilder.toString();
-  }
-
-  /**
-   * Returns the help string for one particular parameter. This respects optional parameters.
-   *
-   * @param parameterType
-   *          (generic) type of the parameter
-   * @param annotations
-   *          annotations of the parameter, may be null
-   * @return string describing the parameter based on source code annotations
-   */
-  private static String getHelpForParameter(Type parameterType, Annotation[] annotations) {
```

Comment:
This is a private method for a single parameter. If someone wants to know what parameters mean they should just go see the javadoc (see my comment above).